helpers/logs.go

```diff
@@ -29,7 +29,7 @@ const (
 	LogActionView   LogAction = iota + 1 // VIEW
 	LogActionAdd                         // ADD
 	LogActionUpdate                      // UPDATE
-	LogActionDelete                      // DELETE
+	LogActionDelete
```

Comment:
I'm guessing this is an accidental change.

Reply:
Yes, thanks.

Reply:
Updated.